pipelines/presidio-golang-deps-CI.yaml

```diff
@@ -0,0 +1,27 @@
+trigger: # Build triggers
```

Comment:
looks good. however, now we have in the base code the ci definition.... 
which is the first time we save it in the source itself. for example presidio's main ci is not saved in this manner...